terminus/theme.py

```diff
@@ -67,6 +70,46 @@ def run(self, theme=None, remove=False, force=False):
 
         elif theme == "default":
             variables = {}
+        elif theme == "adaptive":
+            palette = sublime.ui_info()["color_scheme"]["palette"]
+            gray = "#888888"
+            view = sublime.active_window().active_view()
+            if view:
+                comment_foreground = view.style_for_scope("comment")["foreground"]
+                r = int(comment_foreground[1:3], 16)
+                g = int(comment_foreground[3:5], 16)
+                b = int(comment_foreground[5:7], 16)
+                _, _, s = rgb_to_hls(r/255, g/255, b/255)
+                if s < 0.2:
+                    gray = comment_foreground
+            color_template = "color({} l(- 15%))"
+            red = color_template.format(palette["redish"])
```

Comment:
perhaps we should make the light colors brighter rather than making the regular colors darker?

Reply:
Not sure which one is the better option, but I can change it to make the regular colors darker if you want. I've chosen to make the "normal" colors darker here, because after a quick test it seems that various tools mainly use the "light" colors for their output (at least for me on Windows/PowerShell). Therefore by modifying the "normal" colors, most of the colored output directly matches the original colors from the color scheme. `git diff` as a counter example seems to use the normal colors. Alternatively, we could also decide based on whether the color scheme has dark or light background, to ensure to always have an increased contrast for the modified colors (i.e. make light colors brighter on dark schemes and make normal colors darker on light schemes).

Here are a few screenshot where I set all "light" colors to `#ff00ff` (magenta) to see where these colors are used:

![terminus1](https://user-images.githubusercontent.com/6579999/122778405-4264a780-d2ad-11eb-9045-ad5aed281b5c.png)

![terminus2](https://user-images.githubusercontent.com/6579999/122778425-47c1f200-d2ad-11eb-9aee-5279c415f204.png)

![terminus3](https://user-images.githubusercontent.com/6579999/122778459-4e506980-d2ad-11eb-9edd-732b0370921e.png)

Reply:
I think it is because powershell/windows terminal brighten bold text because they do not support bold text: https://github.com/microsoft/terminal/issues/109

Reply:
done.= d0c465e

Reply:
Perhaps we should enable "brightening bold text" by default? (but there is a chance that we confuse current user)

Reply:
> powershell/windows terminal brightens bold text because they do not support bold text

Oh that's good to know. Alright, I will brighten the light colors instead.